clang/lib/Sema/SemaTemplate.cpp

```diff
@@ -3100,6 +3118,7 @@ BuildDeductionGuideForTypeAlias(Sema &SemaRef,
         Context.getInjectedTemplateArg(NewParam));
     TransformedDeducedAliasArgs[AliasTemplateParamIdx] = NewTemplateArgument;
   }
+  unsigned UndeducedTemplateParameterStartIndex = FPrimeTemplateParams.size();
```

Comment:
Nit, this is a bit of a mouth full:
```suggestion
  unsigned FirstUndeducedParamIdx = FPrimeTemplateParams.size();
```

Reply:
Done.